doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -154,3 +272,133 @@ Here is the offline configuration tool workflow:
    :align: center
 
    offline tool workflow
+
+.. _acrn_config_tool_ui:
+
+How to use ACRN config app
+**************************
+ACRN config app is used to read board info, configure and validate scenario settings,
+generate board source code and scenario source code, configure and validate launch
+settings, and generate launch scripts.
+
+Prerequisites
+=============
+.. _get acrn repo:
+   https://projectacrn.github.io/latest/getting-started/building-from-source.html#get-the-acrn-hypervisor-source-code
+
+- Follow the :ref:`instruction <getting-started-building>` to install the
+  ACRN hypervisor dependencies and tools on your development host.
+
+- Follow the `get acrn repo`_ to download ACRN hypervisor repo to your host.
+
+- Install ACRN config app dependencies:
+
+  .. code-block:: console
+
+     $ cd ~/acrn-hypervisor/misc/acrn-config/config_app
+     $ sudo pip3 install -r requirements
+
+How to use ACRN config app
+==========================
+#. Launch ACRN config app:
+
+   .. code-block:: console
+
+      $ python3 app.py
+
+#. The browser should be launched and navigated to the website:
```

Comment:
make sure you can connect to open network from browser because the app needs to download some javascript files